std/variant.d

```diff
@@ -2204,6 +2247,13 @@ private auto visitImpl(bool Strict, VariantType, Handler...)(VariantType variant
                         result.indices[tidx] = dgidx;
                     }
                 }
+                else static if (isSomeFunction!(dg!T))
+                {
+                    assert(result.generalFuncIdx == -1 ||
+                           result.generalFuncIdx == dgidx,
+                           "Only one generic visitor function is allowed");
```

Comment:
Yep, that `assert` can't be static because `generalFuncIdx` is a 'runtime' value, but the entire function gets run with CTFE, so it will actually fail at compile time if you have > 1 generic visitor (see the tests).